addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php

```diff
@@ -279,9 +279,26 @@ public function index($category = null, $city = null)
 
         $viewType = $this->requestHttp->cookie('viewType');
 
+        if (!isset($allCats)) {
+            if (count($mainCats) == 1 || count($mainCats) == 2) {
+                $catText = end($mainCats)['val'];
+            } elseif (count($mainCats) > 2) {
+                $catArray = array_slice($mainCats, 2);
+                $catText = '';
+                $loop = 0;
+                foreach ($catArray as $cat) {
+                    $catText = !$loop ? $catText . $cat['val'] : $catText . ' ' . $cat['val'];
+                    $loop++;
+                }
+            }
+            $this->template->set('notShowTitle', true);
```

Comment:
showtitle = false could be easy to understand @diashalabi  :)

Reply:
I know I thought about it 🙂 but the way I did it is to check if notShowTitle variable exists, so if it exists and its true we will not show the title, if it doesn't exist we will show the title, we can make it showTitle but it will take extra line of code to check if it exists and assign a default value if it doesn't, I will change it, it looks better 😅